ecosystem/typescript/sdk/src/token_types.ts

```diff
@@ -46,4 +46,6 @@ export interface Token {
   id: TokenId;
   /** server will return string for u64 */
   amount: U64;
+  /** the property map of the token */
+  token_properties: any;
```

Comment:
Can token_properties be defined as `AnyObject` (defined in utils/misc)

Reply:
updated